#!/usr/bin/env python
PACKAGE = "alarm"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("alarm_zone_front_lat", double_t, 0, "Lateral alarm zone distance", 0.3, 0, 1)
gen.add("alarm_zone_front_long", double_t, 0, "Longitudinal alarm zone distance", 0.4, 0, 2)
gen.add("x_offset", double_t, 0, "X offset of the alarm zone center", 0.0, 0, 3)
gen.add("y_offset", double_t, 0, "Y offset of the alarm zone center", 0.0, 0, 4)

exit(gen.generate(PACKAGE, "alarm", "ObstacleDetection"))

